tests/test_configuration.py

```diff
@@ -40,8 +40,6 @@
 
 [strawberryfields.global]
 hbar = 1
-shots = 1000
-analytic = true
```

Comment:
TOML doesn't seem to have the concept of `None` ([see here](https://github.com/toml-lang/toml/issues/30)), so just leaving `shots` out instead of `shots=None`.

Reply:
Good catch!